cypress/integration/demo_tests/demo_spec.js

```diff
@@ -60,4 +60,66 @@ describe('opening OpenSearch-Dashboards', () => {
         cy.get('[data-attr-field="category"]').click()
         cy.get('[data-test-subj="fieldToggle-category"]').click()
     })
+
+    it('Explore the data in the dashboard', () => {
+        /**
+         * Known issue: Comboboxes UI elements are behave inconsistently,
```

Comment:
Bare my English. But is `behave` a verb?

Reply:
You are correct, there is a grammatical mistake there and it would either need to be either `elements are behaving inconsistently` or `elements behave inconsistently`. I will change it to `are behaving`.